com.cubrid.common.ui/src/com/cubrid/common/ui/query/dialog/SetFileEncodingDialog.java

```diff
@@ -159,11 +159,9 @@ protected void constrainShellSize() {
 		if (isOpened) {
 			getShell().setText(Messages.titleOpenFileDialog);
 			setTitle(Messages.titleOpenFileDialogDetail);
-			setMessage(Messages.msgOpenFileDialogDetail);
```

Comment:
In summary, some redundant texts may help novice users to understand the function. So, I think it is desirable not to remove the texts. Korean texts are very helpful, but, english texts may be wrong in some cases and should be fixed or modified appropriately later.